src/i18n/es.json

```diff
@@ -0,0 +1,137 @@
+    "label_email": "Email: ",
+    "label_firstname": "Nombre: ",
+    "label_lastname": "Apellido: ",
+    "label_password": "Contraseña: ",
+    "label_phone": "Teléfono: ",
+    "legal_HTML": "<p>Doppler te informa que los datos de carácter personal que nos proporciones al rellenar el presente formulario serán tratados por Making Sense LLC (Doppler) como responsable de esta web.</p>\n<p><strong>Finalidad:</strong> Darte de alta en nuestra plataforma y brindarte los servicios que nos requieras.</p>\n<p><strong>Legitimación:</strong> Consentimiento del interesado.</p>\n<p><strong>Destinatarios:</strong> Tus datos serán guardados por Doppler, Zoho como CRM, Google como proveedor del servicio de reCAPTCHA, Digital Ocean, Cogeco Peer1 y Rackspace como empresas de hosting.</p>\n<p><strong>Información adicional:</strong> En la <a target=\"_blank\" href=\"https://www.fromdoppler.com/legales/privacidad\">Política de Privacidad</a> de Doppler encontrarás información adicional sobre la recopilación y el uso de su información personal por parte de Doppler, incluida información sobre acceso, conservación, rectificación, eliminación, seguridad, transferencias transfronterizas y otros temas.</p>",
+    "log_in": "Ingresa",
+    "no_more_resend": "¿Aún no has recibido el Email? Ya te lo hemos reenviado, si no llega en los próximos minutos, por favor contáctate con soporte",
+    "placeholder_email": "Tu Email será tu nombre de usuario",
+    "placeholder_firstname": "Nombre",
+    "placeholder_lastname": "Apellido",
+    "placeholder_password": "Escribe tu clave secreta",
+    "placeholder_phone": "Teléfono",
+    "privacy_policy_consent_HTML": "Acepto la <a target=\"_blank\" href=\"https://www.fromdoppler.com/legales/privacidad\">Política de Privacidad</a> de Doppler.",
+    "promotions_consent": "Acepto recibir las promociones de Doppler y sus aliados.",
+    "resend_email": "Reenvíalo",
+    "sign_up": "Regístrate",
+    "sign_up_sub": "Crea una cuenta GRATIS hasta 500 Suscriptores.",
+    "thanks_for_registering": "Gracias por registrarte"
+  },
+  "upgradePlanForm": {
+    "message_placeholder": "Tu mensaje",
+    "plan_select": "Selecciona el Plan",
+    "title": "Solicita una actualización de tu plan"
+  },
+  "validation_messages": {
+    "error_account_is_blocked_invalid_pass_HTML" : "Por seguridad hemos bloqueado tu cuenta momentáneamente. <a href=\"http://www.fromdoppler.com/contacto\" target=\"_blank\">Contáctanos.<a>",
+    "error_account_is_canceled_HTML" : "Tu cuenta se encuentra cancelada. Para más información <a target=\"_blank\" href=\"http://www.fromdoppler.com/contacto\">contáctanos</a>.",
+    "error_checkbox_policy": "¡Ouch! No has aceptado la Política de Privacidad de Doppler.",
+    "error_email_already_exists": "¡Ouch! Ya posees una cuenta en Doppler.",
+    "error_invalid_domain_to_register_account": "¡Ouch! Email inválido para crear una cuenta.",
+    "error_invalid_email_address": "¡Ouch! El formato del email es incorrecto",
```

Comment:
`Email`

Reply:
@carodipietro esto que comenta @cbernat no es parte de los cambios de @GusBaamonde, pero me parece que los comentarios de Cecilia son correctos, ¿Crees que deberíamos aplicar esta sugerencia?

Reply:
@cbernat is right.